data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,411 @@ mission "Ringworld Debris: Quarg"
+			choice
+				`	"How long do you think it will it take?"`
+				`	"Would it be safe to remove our helmets after that?"`
+					goto dumb
+			`	"I have no idea. Usually it is a matter of a few minutes, but we know next to nothing about this tech. We will have to wait and see. In the meanwhile," he says as he begins to activate another scanner, "I can start analyzing the atmosphere that is starting to fill this room. It should save us some time to do it here, as well as also safer, should it be dangerous to our suits."`
+				goto scan
+			label dumb
+			`	Dusk gives you an odd look. "Of course not <first>; it would almost certainly kill you. Such an alien atmosphere is unlikely to be breathable for a human, and even if it was we couldn't be sure until we analyze it extensively." He begins to activate another scanner, "Speaking of which, I can start analyzing the atmosphere that is starting to fill this room. It should save us some time to do it here, as well as also safer, should it be dangerous to our suits."`
+			label scan
+			`	Dusk goes on working on the atmospheric analyzer for about five minutes without nothing happening. You try to connect to the scanner on your ship to check the position of the nearby Fetri'sei, but the dome seems to jam all the communications with the outside. Suddenly, you hear a loud "pop" and the door that leads inside the dome recesses a few centimeters. "Just in time," Dusk says. "My analysis of the atmosphere has finished. Let us go take a look inside." After a brief look at the writing on the door, he grabs a handle on it and slides it to the left. The hatch starts to slowly open inwards. When you come closer to the entrance you are instantly blinded by a strong light coming from the other side of the door. After lowering the sun shield on your helmet, your eyes start adapting to the new light level, and it takes you a few seconds to realize what is now in front of you. Landed inside the cupola, right in front of the airlock, is a Fetri'sei, pointing in the direction of you and Dusk.`
+			`	Both you and Dusk stand still in front of it without saying a word. Out of the corner of your eye, you can see his hands shivering. All of a sudden, the Ka'sei starts emitting a very high pitched sound, very hard to hear, frequently changing in intensity. This lasts only for a few seconds, then abruptly stops.`
+			choice
+				`	(Run back in the airlock.)`
+					goto run
+				`	(Stay still and wait.)`
+			`	The whole cupola falls silent. What feels like hours pass before the Ka'sei swiftly lifts off in front of you, hovering a few meters above the terrain. Just like the previous time, it starts emitting the same sequence of sounds as the last time, followed by a clicking sound. Dusk turns towards you.`
+			choice
+				`	(Run back in the airlock.)`
```

Comment:
```suggestion
				`	(Run back in the airlock.)`
					goto run
```

Reply:
This was actually intentional, only running away at the first choice allows you to go back fast enough to unlock the best outcome (i.e. you are alive and Dusk saves her atmospheric recordings) while every other choice gives you either the middle outcome (atmospheric data destroyed) or kills you

Reply:
Gotcha! Feel free to resolve.